src/frontend/sections/auth/Login.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <h1>Sign In</h1>
+  <p class="text-medium-emphasis">Welcome back! Let's get started</p>
+
```

Comment:
I think we need a "non_field_errors" handler here if ya have bad pass -- I have this one already on confirm password screen:

```
  <v-alert v-if="errors.non_field_errors" type="error">
    <ul>
      <li v-for="(error, index) in errors.non_field_errors" :key="index">{{ error }}</li>
    </ul>
  </v-alert>
```